### PR TITLE
Fab 2646

### PR DESCRIPTION
**Pull request by akprasad-cs**
example for header propagation in agents with chained skills

### CODE REVIEW

**Review by dvasani-CS — APPROVED** [review]

**Review by pkandarpa-cs — APPROVED** [review]

File: propagate-headers-agent-to-skill/skills/chained-daemon-skills/skill-1/requirements.txt [hunk 1]
```diff
@@ -0,0 +1,3 @@
+cortex-python
```
Comment (pkandarpa-cs) [inline propagate-headers-agent-to-skill/skills/chained-daemon-skills/skill-1/requirements.txt:1]: do you need to pin these to specific versions? (I think yes)

File: propagate-headers-agent-to-skill/skills/chained-daemon-skills/skill-2/Makefile [hunk 2]
```diff
@@ -0,0 +1,34 @@
+IMAGE_TAG=$(shell git describe --long --always --match='v*.*' | sed 's/v//; s/-/./')
+SKILL_NAME=$(notdir $(CURDIR))
+DOCKER_IMAGE_URL=${DOCKER_PREGISTRY_URL}/${SKILL_NAME}:${IMAGE_TAG}
+URL=$(shell cat ~/.cortex/config | jq .profiles[.currentProfile].url)
```
Comment (pkandarpa-cs) [inline propagate-headers-agent-to-skill/skills/chained-daemon-skills/skill-2/Makefile:4]: Make sure jq is mentioned in the pre-requisites of the README

File: propagate-headers-agent-to-skill/skills/chained-daemon-skills/skill-2/main.py [hunk 3]
```diff
@@ -0,0 +1,10 @@
+from fastapi import FastAPI
+from fastapi import Request
+
+app = FastAPI()
+
+@app.post('/invoke')
+async def run(request: Request):
+    request_body = await request.json()
+    request_headers = dict(request.headers)
+    return {'payload': request_body}
```
Comment (pkandarpa-cs) [inline propagate-headers-agent-to-skill/skills/chained-daemon-skills/skill-2/main.py:10]: last line should be a new line

File: propagate-headers-agent-to-skill/skills/chained-daemon-skills/skill-2/requirements.txt [hunk 4]
```diff
@@ -0,0 +1,3 @@
+cortex-python
+fastapi
```
Comment (pkandarpa-cs) [inline propagate-headers-agent-to-skill/skills/chained-daemon-skills/skill-2/requirements.txt:2]: pin versions

**Comment by pkandarpa-cs** [comment]
I think you should update the README for this set of examples to have descriptions of all of the variants. JQ is missing from pre-requisites in the README

**Pull request merged**